Description: Disable warnings as errors for jni
 Remove the default use of -Werror when building the jni native
 extensions as expecting a warning-free build of cross-platform
 software is a bit of a fairy tale really.
Reviewed-By: Tim Potter <tpot@hp.com>
Last-Update: 2015-02-25

--- jffi-1.2.7.orig/jni/GNUmakefile
+++ jffi-1.2.7/jni/GNUmakefile
@@ -64,7 +64,7 @@ JFLAGS = -fno-omit-frame-pointer -fno-st
 OFLAGS = -O2 $(JFLAGS)

 # MacOS headers aren't completely warning free, so turn them off
-WERROR = -Werror
+WERROR =
 ifneq ($(OS),darwin)
   WFLAGS += -Wundef $(WERROR)
 endif
